vcxagency-node/config/localhost-tls.env

```diff
@@ -0,0 +1,27 @@
+LOG_LEVEL='debug'
```

Comment:
as this file only differs from `localhost.env` by change in `SERVER_ENABLE_TLS=true` I'd suggest to remove this